README.md

```diff
@@ -12,7 +12,7 @@ for any video conferencing, live-streaming, or broadcasting solutions where you
 However, you can also use it for non-real-time use cases, for example, apply some effect on a video from an MP4 file and write the output
 to the new MP4.
 
-We don't have plans to introduce any major breaking changes in the API in the foreseeable future.
+https://github.com/user-attachments/assets/d8716866-67bd-4721-b1c2-1a1d31e1c5b5
```

Comment:
can we link that directly from this repo?

Reply:
Nope. At least I haven't found a was to do it (syntax from including images doesn't work)